HMCL/src/main/resources/assets/lang/I18N_zh_CN.properties

```
@@ -1230,6 +1230,11 @@ terracotta.back=退出
 terracotta.feedback.title=填写反馈表
 terracotta.feedback.desc=在 HMCL 更新联机核心时，我们欢迎您用 10 秒时间填写联机质量反馈收集表。
 terracotta.sudo_installing=HMCL 需要验证您的密码才能安装联机核心
+terracotta.difficulty.easiest=当前网络状态极好：稍等一下就成功！
+terracotta.difficulty.simple=当前网络状态较好：建立连接需要一段时间……
+terracotta.difficulty.medium=当前网络状态中等：已启用抗干扰备用隧道，但需要更长时间（甚至失败）
```

需要改成半角。Glavo 在 Gradle 自动检测规则中添加了这个。

```suggestion
terracotta.difficulty.medium=当前网络状态中等：已启用抗干扰备用隧道，但需要更长时间 (甚至失败)
```